src/main/java/org/openRealmOfStars/game/States/AITurnView.java

```diff
@@ -170,6 +172,7 @@ private void handleMissions(final Fleet fleet, final PlayerInfo info) {
       case ATTACK:
         MissionHandling.handleAttack(mission, fleet, info, game);
         break;
+      default:
```

Comment:
Maybe throw an exception here. Default branch is only selected if new enum is added but not handled in switch case.